assembl/models/post.py

```diff
@@ -631,12 +631,19 @@ def generate_message_id(cls):
         onupdate='CASCADE'
     ), primary_key=True)
 
+    modification_date = Column(DateTime)
```

Comment:
After discussion with MAP, there was to much work to enable HistoryMixin on Post. This is not simply inheriting of HistoryMixin. So we decided to just add a modification_date field. The only fields that are versionned are subject and body via tombstoned langstringentry.